README.md

```diff
@@ -335,7 +335,7 @@ Refer to [ui/app/CONTRIBUTING.md](ui/app/CONTRIBUTING.md).
 
 ## Architecture
 
-![](https://raw.githubusercontent.com/prometheus/alertmanager/4e6695682acd2580773a904e4aa2e3b927ee27b7/doc/arch.jpg)
+![](https://raw.githubusercontent.com/prometheus/alertmanager/master/doc/arch.jpg)
```

Comment:
Instead of having a fully qualified domain name here, why not just referencing the image relative to the repository?(`doc/arch.jpg`). You can see an example here: https://github.com/improbable-eng/thanos/blame/master/README.md#L1

Otherwise the `release-xxx` branch `README.md` would always point to the `master` branch image, which might diverge at some point.

Reply:
Good call, PR updated with relative path in README.md.